dao/src/main/java/com/epam/esm/dao/config/PersistenceConfig.java

```diff
@@ -1,45 +1,14 @@
 package com.epam.esm.dao.config;
 
-import com.zaxxer.hikari.HikariConfig;
-import com.zaxxer.hikari.HikariDataSource;
-import org.springframework.context.annotation.Bean;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
-import org.springframework.core.io.ClassPathResource;
-import org.springframework.core.io.Resource;
-import org.springframework.core.io.support.PropertiesLoaderUtils;
-import org.springframework.jdbc.core.JdbcTemplate;
-import org.springframework.jdbc.datasource.DataSourceTransactionManager;
 import org.springframework.transaction.annotation.EnableTransactionManagement;
 
-import javax.sql.DataSource;
-import java.io.IOException;
-import java.util.Properties;
-
-@Profile("dev")
+@Profile("prod")
```

Comment:
Why have you changed profile here?

Reply:
I suppose I forgot to delete this annotation when I migrated to Spring Boot. The presence of this annotation is not necessary. (fixed)